client/state/selectors/get-current-plan-term.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+
+import { getPlan } from 'lib/plans';
+import { getSitePlan } from 'state/sites/selectors';
+
+/**
+ * Returns term of the active plan for given siteId, e.g. TERM_MONTHLY
+ *
+ * @param {Object} state Current state
+ * @param {Number} siteId Site ID
+ * @return {string} Human-readable interval type
+ */
+export default function getTermFromCurrentPlan( state, siteId ) {
```

Comment:
Do we need the named export?

Reply:
There's only a default export here, it just happens to be a named function 😉